examples/input/gamepad_demo.rs

```diff
@@ -0,0 +1,519 @@
+//! Shows a visualization of gamepad buttons, sticks, and triggers
+
+use std::f32::consts::PI;
+
+use bevy::{
+    input::gamepad::{GamepadButton, GamepadSettings},
+    prelude::*,
+    sprite::{MaterialMesh2dBundle, Mesh2dHandle},
+};
+
+const BUTTON_RADIUS: f32 = 25.;
+const BUTTON_CLUSTER_RADIUS: f32 = 50.;
+const START_SIZE: Vec2 = Vec2::new(30., 15.);
+const TRIGGER_SIZE: Vec2 = Vec2::new(70., 20.);
+const STICK_BOUNDS_SIZE: f32 = 100.;
+
+const BUTTONS_X: f32 = 150.;
+const BUTTONS_Y: f32 = 80.;
+const STICKS_X: f32 = 150.;
+const STICKS_Y: f32 = -135.;
+
+const NORMAL_BUTTON_COLOR: Color = Color::rgb(0.2, 0.2, 0.2);
+const ACTIVE_BUTTON_COLOR: Color = Color::PURPLE;
+const LIVE_COLOR: Color = Color::rgb(0.4, 0.4, 0.4);
+const DEAD_COLOR: Color = Color::rgb(0.3, 0.3, 0.3);
+const EXTENT_COLOR: Color = Color::rgb(0.3, 0.3, 0.3);
+const TEXT_COLOR: Color = Color::WHITE;
```

Comment:
I'm open to alternate color schemes. I feel like it would be best to match existing examples, especially with the bevy examples page on the website though.

I'm curious to hear other opinions about this, but the proposed set looks a little out of place to me, and the contrast between full-white text and full-black background seems a bit intense.

<img width="1220" alt="image" src="https://user-images.githubusercontent.com/200550/191975967-494f0fb6-5aa6-4740-a0a0-b4eae295e896.png">

Reply:
Yeah, I'd prefer keeping the default bevy clear color and use colors similar to other examples